requirements.txt

```diff
@@ -1,9 +1,7 @@
 Authlib==1.0.1
-cryptography==37.0.2
 Django==3.2.13
 django-csp==3.7
-git+https://github.com/cal-itp/eligibility-api#egg=eligibility_api
+git+https://github.com/cal-itp/eligibility-api@a2f9932d074d10afc21713ada32eebe3841a1a3d#egg=eligibility_api
```

Comment:
This hash can be removed once [cal-itp/eligibility-api#31](https://github.com/cal-itp/eligibility-api/pull/31) is merged into the `main` branch over in that repo.